app/controllers/application_controller.rb

```diff
@@ -103,7 +103,9 @@ def strip_nested_param_whitespaces(object)
   end
 
   def replace_devise_notice_flash_with_success!
-    flash[:success] = flash.discard(:notice) if flash[:notice].present?
+    return unless flash[:notice].present?
+
+    flash[:success] = flash.discard(:notice)
```

Comment:
This just looks nicer to me with a guard clause.

Reply:
to me it was more readable the way it was before, but it is personal taste. I can live with this

Reply:
I hope you don't mind.